### DISCUSSION TITLE
v3 and mobx compatibility

### DISCUSSION THREAD

**Discussion by mikael-floden** [infinum/datx#1201] [v3]
Will it still be possible to run datx with mobx after upgrading to v3?

We are running Angular and never disabled mobx (since we like the power and simplicity it gave us). What will happen when we upgrade to v3? Can we still use mobx autorun if we manually call  makeAutoObservable on all models returned from the collection?

The combination datx + mobx was so powerful.

If v3 is still compatible with mobx, all help we can get on how to best restore the integration is appreciated!

**Comment by DarkoKukovec** [comment]
Hi, the primary purpose of v3 is to remove mobx and streamline the code a bit, since we don't use it on our newer projects, so the answer is no.
However, we still have some mobx projects (with no plan to migrate to v3), so v2 will be maintained for the foreseeable future, and there is no need to migrate to v3 if you're using mobx.

**Reply by mikael-floden** [reply]
Thanks for the answer, we will stay at v2 then
